src/ui/components/Collections/Collections.jsx

```diff
@@ -143,7 +143,9 @@ const Collections = props => {
         headers = [
           { value: 'id', label: 'id', edit: false },
           { value: 'fullUrl', label: 'fullUrl', edit: true },
-          { value: 'criteria', label: 'criteria', edit: true },
+          { value: 'criteria', label: 'criteria', edit: false },
```

Comment:
`edit: false` doesn't mean you can't define it when you make the resource.  If you click "add new" and make a new subscription, you can define the criteria but it gets put at the top level, instead of in the `resource`.  This would be a `viewOnly` field like the server icon:

`{ value: 'icon', label: '', edit: false, viewOnly: true }`